src/bot.js

```diff
@@ -254,6 +254,20 @@ client.on(Events.MessageCreate, async message => {
 		let userInput = message.content
 			.replace(new RegExp("^\s*" + myMention.source, ""), "").trim();
 
+		// Process text file if attached
+        if (message.attachments.size > 0) {
+            const txtAttachment = message.attachments.find(att => att.name.endsWith('.txt'));
```

Comment:
I'm not sure how this would work with language models that can also recognise image/video files, this could be tweaked to work with those too. Would it be possible for the file to sent for ollama to deal with?